htdocs/cron/class/cronjob.class.php

```diff
@@ -42,38 +47,111 @@ class Cronjob extends CommonObject
+		"unitfrequency" => array("type"=>"varchar(255)", "label"=>"Unitfrequency", "enabled"=>"1", 'position'=>110, 'notnull'=>1, "visible"=>"-1", "alwayseditable"=>"1",),
+		"frequency" => array("type"=>"integer", "label"=>"Frequency", "enabled"=>"1", 'position'=>115, 'notnull'=>1, "visible"=>"-1", "alwayseditable"=>"1",),
+		"nbrun" => array("type"=>"integer", "label"=>"Nbrun", "enabled"=>"1", 'position'=>120, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+		"status" => array("type"=>"integer", "label"=>"Status", "enabled"=>"1", 'position'=>500, 'notnull'=>1, "visible"=>"-1", "alwayseditable"=>"1",),
+		"fk_user_author" => array("type"=>"integer:User:user/class/user.class.php", "label"=>"Fkuserauthor", "enabled"=>"1", 'position'=>130, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1", "css"=>"maxwidth500 widthcentpercentminusxx", "csslist"=>"tdoverflowmax150",),
+		"fk_user_mod" => array("type"=>"integer:User:user/class/user.class.php", "label"=>"Fkusermod", "enabled"=>"1", 'position'=>135, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1", "css"=>"maxwidth500 widthcentpercentminusxx", "csslist"=>"tdoverflowmax150",),
+		"note" => array("type"=>"text", "label"=>"Note", "enabled"=>"1", 'position'=>140, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+		"libname" => array("type"=>"varchar(255)", "label"=>"Libname", "enabled"=>"1", 'position'=>145, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+		'entity' =>array('type'=>'integer', 'label'=>'Entity', 'default'=>1, 'enabled'=>1, 'visible'=>3, 'notnull'=>1, 'position'=>30, 'index'=>1),
+		"maxrun" => array("type"=>"integer", "label"=>"Maxrun", "enabled"=>"1", 'position'=>155, 'notnull'=>1, "visible"=>"-1", "alwayseditable"=>"1",),
+		"autodelete" => array("type"=>"integer", "label"=>"Autodelete", "enabled"=>"1", 'position'=>160, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+		"fk_mailing" => array("type"=>"integer", "label"=>"Fkmailing", "enabled"=>"1", 'position'=>165, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1", "css"=>"maxwidth500 widthcentpercentminusxx",),
+		"test" => array("type"=>"varchar(255)", "label"=>"Test", "enabled"=>"1", 'position'=>170, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+		"processing" => array("type"=>"integer", "label"=>"Processing", "enabled"=>"1", 'position'=>175, 'notnull'=>1, "visible"=>"-1", "alwayseditable"=>"1",),
+		"email_alert" => array("type"=>"varchar(128)", "label"=>"Emailalert", "enabled"=>"1", 'position'=>180, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+		"pid" => array("type"=>"integer", "label"=>"Pid", "enabled"=>"1", 'position'=>185, 'notnull'=>0, "visible"=>"-1", "alwayseditable"=>"1",),
+	);
+	public $rowid;
+	public $tms;
+	public $datec;
+	public $jobtype;
 	public $label;
-
-	/**
-	 * @var string Job command
-	 */
 	public $command;
```

Comment:
@moreauftheobald what is the reason to remove all phpdoc ?

Reply:
i've done the refactoring using module builder witch dont provide phpdoc on this point. so i thout it was not mandatory .
if needed i'll put them back ....

just told me.